app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RatingsController < ApplicationController
+  def new
+    @rating = Rating.new
+    @user_id = params['user_id']
+  end
+
+  def create
+    Rating.create(user_id: params["user_id"], comment: params["rating"]["comment"])
+    User.find(params['user_id']).update_attribute(:status, :certified)
```

Comment:
I think this is supposed to be in a different action called update and not create